src/acl/Acl.cc

```diff
@@ -291,8 +291,20 @@ ACL::isProxyAuth() const
 void
 ACL::parseFlags()
 {
-    // ACL kids that carry ACLData which supports parameter flags override this
-    Acl::ParseFlags(options(), Acl::NoFlags());
+    Acl::Options allOptions = options();
+    for (const auto opt: lineOptions())
+        allOptions.insert(opt);
+    Acl::ParseFlags(allOptions);
+}
+
+const Acl::Options &
+ACL::lineOptions()
+{
+    if (auto lOptions = dirtyLineOptions()) {
```

Comment:
If possible:
```suggestion
    if (const auto lOptions = dirtyLineOptions()) {
```

Reply:
Just FYI: That non-constant reset() call does not modify the pointer called `lOptions`. No need to change this right now because I think this code will have to be refactored (for more important reasons) anyway.